.github/scripts/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "validator-runner-tests",
+  "private": true,
+  "type": "commonjs",
+  "scripts": {
+    "test": "jest --runInBand"
+  },
+  "devDependencies": {
+    "jest": "^29.7.0"
```

```suggestion
    "vitest": "^3.1.2"
```

We are using vitest instead of jest for all new tests going forward (and will be converting older projects to vitest eventually).